filters/ELMFilter.hpp

```diff
@@ -58,9 +58,23 @@ class PDAL_DLL ELMFilter : public Filter
     double m_cell;
     double m_threshold;
     uint8_t m_class;
+    bool m_maximum;
+
+    Dimension::Id m_Zdim;
```

Comment:
These names don't follow our normal convention.  It should probably be "m_zDim", "m_classDim", etc.

Reply:
I also wonder how meaningful it is to offer an alternative to Z. Did you have another dimension in mind @hobu? Intensity? If we need to retain it, we can call it `m_fltDim` or similar.